docs/deployment/deploy_gardenlet.md

```diff
@@ -0,0 +1,344 @@
+Hence, the `Soil` cluster hosts the control planes of the other `Seed` clusters.
+
+The Gardenlet deployed in the `Soil` cluster, can deploy itself into the `Shooted Seed` clusters.  
+The advantage of this approach is, that there is only one initial Gardenlet installation required - every other `Shooted Seed` has a Gardenlet deployed automatically.
+
+## Prerequisites
+
+- **Kubernetes cluster that should be registered as a Seed cluster**
+    
+    - The minimum required Kubernetes version is `1.11`.
+    - Determine the nodes, pods and services CIDR of the Cluster. This needs to be configured in the `Seed` configuration. Gardener uses this information to check that the Shoot cluster is not created with overlapping CIDR ranges.
+    - A current prerequisite of Kubernetes clusters that are used as Seed clusters is to have a pre-deployed Ingress controller. This is pre-installed for `Shooted Seed` clusters.  
+        An ingress controller is required to:
+        - configure the `Seed` cluster resource (tells the Gardenlet which ingress domain it can use to deploy ingress resources for seed components like grafana and prometheus).
+            
+        - handle ingress resources that are deployed during the Seed bootstrapp by the Gardenlet.
+            
+            :warning:  
+            There should exist a DNS record `*.ingress.<SEED-CLUSTER-DOMAIN>` where `<SEED-CLUSTER-DOMAIN>` is the value of the `.dns.ingressDomain` field of [a Seed cluster resource](/Applications/Joplin.app/Contents/example/50-seed.yaml "../../example/50-seed.yaml") (or the [respective Gardenlet configuration](/Applications/Joplin.app/Contents/example/20-componentconfig-gardenlet.yaml#L84-L85 "../../example/20-componentconfig-gardenlet.yaml#L84-L85")).
+            
+            **This is how it could be done for the Nginx ingress controller**
+            
+            Deploy nginx into the `kube-system` namespace in the Kubernetes cluster that should be registered as a `Seed`.
+            
+            Nginx will on most cloud providers create the service with type loadbalancer with an external ip.
+            
+            ```
+            NAME                        TYPE           CLUSTER-IP    EXTERNAL-IP
+            nginx-ingress-controller    LoadBalancer   10.0.15.46    34.200.30.30
+            ```
+            
+            Create an wildcard `A` record (e.g *.ingress.sweet-seed.<my-domain>. IN A 34.200.30.30) with your DNS provider and point it to the external ip of the ingress service. This ingress domain is later required to register the `Seed` cluster.
+            
+- **Kubeconfig for the Seed cluster**
+    
+    - Needs full admin rights
+    - Needs to be stored as a secret in the Seed cluster (configured by Gardenlet configuration `seedClientConnection.kubeconfig`)
+- **Cloud Provider credentials (e.g. AWS) for the Seed cluster**
+    
+    - Only required when the Seed should be automatically registered by the Gardenlet
```

Comment:
Hm, why is this listed here then? This documentation is not about this use-case, right? It follows further down?